### PR TITLE
Docs updates

### PR DESCRIPTION
**Pull request by iamdefinitelyahuman**
### What I did
1. Expand the testing documentation to include examples using [Brownie](https://github.com/iamdefinitelyahuman/brownie)
2. Add a docs build to the CI, fix errors so that it passes

### How I did it
Magic.

### How to verify it
Read the docs.

### Cute Animal Picture

image

### CODE REVIEW

**Comment by iamdefinitelyahuman** [comment]
Did I really just fail my own docs build on it's first run?  Oh, the irony...

File: docs/index.rst [hunk 1]
```diff
@@ -6,7 +6,7 @@ Vyper
     :alt: Vyper logo
     :align: center
 
-Vyper is a contract-oriented, pythonic programming language that targets the `Ethereum Virtual Machine (EVM) <http://ethdocs.org/en/latest/introduction/what-is-ethereum.html#ethereum-virtual-machine>`_
+Vyper is a contract-oriented, pythonic programming language that targets the `Ethereum Virtual Machine (EVM) <https://ethereum-homestead.readthedocs.io/en/latest/introduction/what-is-ethereum.html#ethereum-virtual-machine>`_.
```
Comment (fubuloubu) [inline docs/index.rst]: Can't we link to an article on ethereum.org or ethhub.io?

Comment (iamdefinitelyahuman) [reply docs/index.rst]: one of these maybe?

[Solidity's version](https://solidity.readthedocs.io/en/v0.6.0/introduction-to-smart-contracts.html#index-6) - I think they paraphrased the homestead docs
[the EVM awesome list](https://github.com/ethereum/wiki/wiki/Ethereum-Virtual-Machine-(EVM)-Awesome-List) - a big list, let people find their own resources
[Ethereum in depth](https://blog.openzeppelin.com/ethereum-in-depth-part-1-968981e6f833/) - some good content by zeppelin

Comment (fubuloubu) [reply docs/index.rst]: Maybe something like https://ethereum.org/learn/#ethereum-basics or https://docs.ethhub.io/ethereum-basics/what-is-ethereum/

**Pull request merged**